.vsts-ci.yml

```diff
@@ -141,14 +141,12 @@ jobs:
         TASK: bdist
         PYTHON_VERSION: 3.6
   steps:
-  - task: CondaEnvironment@1
-    inputs:
-      createCustomEnvironment: true
-      updateConda: true
-      environmentName: $(CONDA_ENV)
-      packageSpecs: 'python=$(PYTHON_VERSION) matplotlib nose numpy pandas psutil pytest python-graphviz scikit-learn scipy'
-      createOptions: '-q'
-  - powershell: $(Build.SourcesDirectory)/.ci/test_windows.ps1
+  - powershell: Write-Host "##vso[task.prependpath]$env:CONDA\Scripts"
+    displayName: Enable conda
+  - script: |
+      conda update -q -y conda
+      conda create -q -y -n %CONDA_ENV% python=%PYTHON_VERSION% matplotlib nose numpy pandas psutil pytest python-graphviz scikit-learn scipy
+      cmd /c "activate %CONDA_ENV% & powershell -ExecutionPolicy Bypass -File %BUILD_SOURCESDIRECTORY%/.ci/test_windows.ps1"
```

Comment:
Unfortunately, despite the fact that `conda>=4.6`[ supports environment managing in PowerShell](https://github.com/conda/conda/issues/626), I can't get it work neither on my home machine, nor at CI side. So, I included [this hack](https://stackoverflow.com/a/52237127).

I tested notebook with error, example with error, and, of course, impassable test. All cases expectedly fail a CI job, so, I hope that `LastExitCode` is handled properly here.